Formula/kustomize.rb

```diff
@@ -2,8 +2,8 @@ class Kustomize < Formula
   desc "Template-free customization of Kubernetes YAML manifests"
   homepage "https://github.com/kubernetes-sigs/kustomize"
   url "https://github.com/kubernetes-sigs/kustomize.git",
-      :tag      => "v3.0.2",
-      :revision => "aa2313c2825a7712b12309c1cd7798f371a0bb18"
+      :tag      => "3.0.3",
```

Comment:
I don't know why the `v` was erroneously dropped from the version, nor how this made it out of CI, but [one can see it is v3.0.3](https://github.com/kubernetes-sigs/kustomize/releases) and the typo causes `brew install -s kustomize` to fail

Reply:
yeah, I think the release tag was wrong earlier.